### ISSUE TITLE
Table Class "Compact" Has Poor Column Label Layout

### ISSUE THREAD

**Issue by phillc73** [rstudio/DT#36]
If applying the Table CSS Class "compact" the column order arrows appear underneath the column labels.

Below are two images to demonstrate this. The first image is "normal" table layout. The second image uses class = 'compact'

![tablelayoutnormal](https://cloud.githubusercontent.com/assets/815126/6941778/98da7678-d87a-11e4-85cf-3cd0174c7554.png)
![tablelayoutcompact](https://cloud.githubusercontent.com/assets/815126/6941784/9d49b282-d87a-11e4-8ebe-b0b01f6ff615.png)

Code to produce the table with layout error.

```R
library("shiny")
library("shinydashboard")
library("datasets")
library("DT")

header <- dashboardHeader()

sidebar <- dashboardSidebar()

body <- dashboardBody(
  DT::dataTableOutput("mtcarsTable")
)

shinyApp(
  ui = dashboardPage(header, sidebar, body),
  server = function(input, output) {
    
    output$mtcarsTable <- renderDataTable({
      DT::datatable(datasets::mtcars, 
                    class = 'compact',
                    options = list(rowCallback = JS('
                          function(nRow, aData, iDisplayIndex, iDisplayIndexFull) {
                                                    // Bold and green cells for conditions
                                                    if (parseFloat(aData[3]) >= 200)
                                                    $("td:eq(3)", nRow).css("font-weight", "bold");
                                                    if (parseFloat(aData[3]) >= 100)
                                                    $("td:eq(3)", nRow).css("background-color", "#9BF59B");
                                                     }'),
                                   initComplete = JS(
                                     "function(settings, json) {",
                                     "$(this.api().table().header()).css({'background-color': '#949DA6', 'color': '#000'});",
                                     "}")
                                    )
      )
    })
  }
)
```

**Comment by yihui** [comment]
Sorry, but that is just from the upstream DataTables library. I _might_ be able to tweak the CSS by myself, but I'd prefer it is done upstream. You may report it to https://github.com/DataTables/DataTables or http://datatables.net/forums/ Thanks!

**Comment by phillc73** [comment]
A note to say this has been fixed upstream and will be released in 1.10.6

https://github.com/DataTables/DataTables/issues/527

**Comment by yihui** [comment]
Awesome. Thanks so much for reporting to DataTables! I'll remember to upgrade to 1.10.6 when it is released.
